### PR TITLE
Create Jira Health Dashboard

### PR DESCRIPTION
**Pull request by TimWheeler-SQUP**
Jira health board - contains Web API and PowerShell tiles

### CODE REVIEW

File: dashboards/jira-health/readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## How to use this dashboard
+
+### Prerequisities
+* A Jira cloud instance with at least one project and version in it.
+* **SquaredUp minimum version 5.2** --> [Download](https://download.squaredup.com/)
+
+### Setup the Web API integration
+- Navigate to System > Integrations
+- Create new Web API integration called "Jira"
+- Enter base url for your Jira instance API endpoint: i.e. `https://<your-prometheus-url/api/v1/`
```
Comment (john-knowles) [inline dashboards/jira-health/readme.md:21]: @TimWheeler-SQUP please can you update the URL reference here.

**Review by john-knowles — CHANGES_REQUESTED** [review]

File: dashboards/jira-health/readme.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## How to use this dashboard
+
+### Prerequisities
+* A Jira cloud instance with at least one project and version in it.
+* **SquaredUp minimum version 5.2** --> [Download](https://download.squaredup.com/)
+
+### Setup the Web API integration
+- Navigate to System > Integrations
+- Create new Web API integration called "Jira"
```
Comment (john-knowles) [inline dashboards/jira-health/readme.md:20]: @TimWheeler-SQUP which type of web api integration?

File: dashboards/jira-health/readme.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- **SquaredUp minimum version 5.2** --> [Download](https://download.squaredup.com/)
+
+### Setup the Web API integration
+- Navigate to System > Integrations
+- Create a new 'Basic Auth' Web API integration called "Jira API basic"
+- Enter base url for your Jira instance API endpoint: i.e. `https://<JIRA domain>.atlassian.net/rest/api/3/`
+- Save with appropriate authentication details
+
+### Setup the PowerShell tile
+- Add a profile that contains the details you will send in the Header variable of your API. This will be an encoded version of your username and password pair - https://community.atlassian.com/t5/Jira-questions/JIRA-API-with-Powershell/qaq-p/992343
```
Comment (john-knowles) [inline dashboards/jira-health/readme.md:25]: @TimWheeler-SQUP here could we replace the forum link with a code snippet of the powershell profile (minus any credentials?

**Review by john-knowles — CHANGES_REQUESTED** [review]

File: dashboards/jira-health/readme.md (hunk 3 above)
Comment (TimWheeler-SQUP) [reply dashboards/jira-health/readme.md]: Done

File: dashboards/jira-health/readme.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+$encodedCreds = [System.Convert]::ToBase64String([System.Text.Encoding]::ASCII.GetBytes($pair))
+
+$basicAuthValue = "Basic $encodedCreds"
+
+$Headers = @{
+ Authorization = $basicAuthValue
+}
+```
+Invoke-WebRequest -Uri 'https://<jira_base_url>/rest/api/2/search?jql=project = CIT AND issuetype = "JIRA Request" AND (created >= startOfDay(-30d) OR updated >= startOfDay(-30d))' `
```
Comment (jellyash) [inline dashboards/jira-health/readme.md:39]: Should this sit within the code block above?

Comment (TimWheeler-SQUP) [reply dashboards/jira-health/readme.md:39]: Dammit. Fixed.

**Review by john-knowles — APPROVED** [review]

**Comment by john-knowles** [comment]
Great stuff @TimWheeler-SQUP

**Review by shawn149 — APPROVED** [review]

**Pull request merged**